docker/travis_test/Dockerfile

```diff
@@ -1,11 +1,11 @@
 #!BuildTag: openqa_dev
-FROM opensuse:42.3
+FROM opensuse/leap:15.0
```

Comment:
That is the file for the Docker container we use in our CI to run tests. Since most OSD workers are still on Leap 42.3 (or SLE 12) I wouldn't change that.

Reply:
Leap 42.3 is EOL in June. Maybe time to update to Leap 15.0, or even 15.1 as 15.0 is EOL in November.

Reply:
@ggardet regardless of the EOL please keep the 42.3 for now in `docker/travis_test/Dockerfile` so that our test environment corresponds to the oldest openSUSE version we want to support. There is an openQA-in-openQA test against Tumbleweed in place so it's good to cover a broad span within our tests.

Reply:
@okurz, ok, I reverted this part. So, it should be fine now.